app/panel/utils/utils.js

```diff
@@ -232,12 +233,21 @@ export function setTheme(doc, name, account) {
 		const { css } = themeData[name];
 
 		// Create element for the theme being set, if it is not there
-		const themeStyle = doc.createElement('style');
-		themeStyle.id = name;
-		themeStyle.title = `${styleTitlePrefix} ${name}`;
-
-		// Set content of style element to the theme text.
-		themeStyle.textContent = css;
-		document.head.appendChild(themeStyle);
+		if (!themeStyle) {
+			themeStyle = doc.createElement('style');
+			themeStyle.id = name;
+			themeStyle.title = `${styleTitlePrefix}`;
+			themeStyle.textContent = css;
+			doc.head.appendChild(themeStyle);
+		} else {
+			themeStyle.textContent = css;
+		}
```

Comment:
Simplify by removing the `else` and putting `themeStyle.textContent = css` below the conditional block